pkg/volume/iscsi/iscsi.go

```diff
@@ -189,7 +193,8 @@ func (b *iscsiDiskBuilder) SetUpAt(dir string, fsGroup *int64) error {
 
 type iscsiDiskCleaner struct {
 	*iscsiDisk
-	mounter mount.Interface
+	mounter    mount.Interface
+	deviceUtil ioutil.DeviceUtil
```

Comment:
Looks like `DeviceUtil` only ends up getting consumed through `iscsiDiskBuilder`, so let's not add it to the `iscsiDiskCleaner` if it's not needed.